images/metering-ansible-operator/roles/meteringconfig/tasks/deploy_resources.yml

```diff
@@ -56,4 +56,4 @@
   loop_control:
     loop_var: resource
     label: "{{ resource.template_file }}"
-  when: not (resource.create | default(true))
+  when: not (resource.create | default(true))
```

Comment:
Typically YAML files have a newline at the end of the file - you can configure your editor to automatically append newlines to the end of a file (for applicable file formats).